### ISSUE TITLE
Clever regex to limit Journal days?

### ISSUE THREAD

**Issue by jvonhoff** [joekrill/silverbullet-treeview#17]
I'm curious if anyone knows a clever regex to hide all but the last few entries in the Journal folders?

This was as close as I have gotten thus far, I just have to update it monthly.
![image](https://github.com/user-attachments/assets/604ce3bb-acff-438d-965c-4529d8d9e25b)

**Comment by joekrill** [comment]
@jvonhoff  I wasn't able to come up with a way to do this using regex alone. But I just updated the plug and made some changes to filtering:

There are now 3 ways to filter
1. By regex
2. By tags
3. By a custom space function

Using a custom space function can achieve the functionality you wanted. I used your use-case as an example implementation in the README. [Check it out](https://github.com/joekrill/silverbullet-treeview?tab=readme-ov-file#filtering-by-custom-function-example) and let me know if that works for you.
